apps/demos/testing/widgets/chat/Overview.test.js

```diff
@@ -0,0 +1,31 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+import { testScreenshot } from '../../../utils/visual-tests/helpers/theme-utils';
+
+fixture('Chat.Overview')
+  .page('http://localhost:8080/')
+  .before(async (ctx) => {
+    ctx.initialWindowSize = [900, 800];
+  });
+
+runManualTest('Chat', 'Overview', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
+  test('Overview', async (t) => {
+    const styleElement = document.createElement('style');
+    styleElement.innerHTML = `.dx-chat-typingindicator-circle { animation: none !important; }`;  
+    document.head.appendChild(styleElement);
+
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await t.typeText('#user-chat .dx-texteditor-input', 'testing');
+
+    await testScreenshot(t, takeScreenshot, 'chat_typing_indicator_appears_in_user_chat.png');
+
+    await t.typeText('#support-chat .dx-texteditor-input', 'testing');
```

Comment:
This approach will not work because IDs are not added t Chats in the all frameworks. We should use a common approach to determine a specific Chat element on the page, something like this
`
Selector('.dx-chat').nth(1)
`

Reply:
https://github.com/DevExpress/DevExtreme/pull/28513/commits/3bde3a32c992d44752bf0839478bdfc6e4cb44b3